### ISSUE TITLE
error on baseline

### ISSUE THREAD

**Issue by akevalion** [tinchodias/FFICallLogger#1]
I used the next script to load the project on pharo10 image macOs monterrey
```st
Metacello new
  baseline: 'DataFrame';
  repository: 'github://PolyMathOrg/DataFrame/src';
  load.
Metacello new
    baseline: 'FFICallLogger';
    repository: 'github://tinchodias/FFICallLogger';
    load.

TFLCallLogger install.

t := TFLCallsPerSecondTracer new.
t start.
t stop
```

But I got an error, related to DataFrameMarkDownPrinter.

The baseline should load all dependencies

**Comment by akevalion** [comment]
@tinchodias could you please review this 
<img width="136" alt="image" src="https://user-images.githubusercontent.com/10532890/160573166-8d3f65da-b2bb-4f9c-9cb0-5ad787eb6001.png">

**Comment by tinchodias** [comment]
thanks Milton, I'll check it ASAP

**Comment by tinchodias** [comment]
I just realized that DataFrameMarkDownPrinter is in other project, and it's not enough to add that baseline as dependency.

**Comment by tinchodias** [comment]
Fixed with f5af463c780d012815387f04566a675f5a2c6f79
